src/task.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";

```

Comment:
The file name should be 'Task' not 'task'

The convention for components names is to be capitalized